runhouse/servers/env_servlet.py

```diff
@@ -297,11 +336,26 @@ def _status_local_helper(self):
 
         # Try loading GPU data (if relevant)
         env_gpu_usage = (
-            self._get_env_gpu_usage(int(env_servlet_pid))
-            if cluster_config.get("has_cuda", False)
-            else {}
+            self._get_env_gpu_usage() if cluster_config.get("has_cuda", False) else {}
```

Comment:
instead of storing in the config can't we reuse the logic to check if cuda is enabled on the cluster?